src/style.css

```diff
@@ -0,0 +1,320 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  height: calc(100% - 20px);
+  overflow-y: auto;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  justify-content: space-around;
+  padding-top: 2%;
+}
```

Comment:
- [ ] I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)

In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
```

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓

Reply:
resolved